### PR TITLE
59 app tile

### PR DESCRIPTION
**Pull request by FearsomeRover**
I added the konf app tile, and replaced the counter with it. I know, it wont be used for a while. 
magyar
angol
I think the english buttons look way better.

### CODE REVIEW

File: src/components/tiles/mobil-app-tile.tsx
Comment (Tschonti) [inline src/components/tiles/mobil-app-tile.tsx]: while you're there, change the alt text

**Review by Tschonti — CHANGES_REQUESTED** [review]
The google image behaves differently than the apple one. Is there no svg version of the google image?
image

I agree that the English buttons look better, but everything else is in Hungarian, so I think we should stick with that.

Also, it'd be great if we could include the questions in the text. What do you think about this: 
Töltsd le a mobil appot, hogy élőben köveshesd a konferencia eseményeit és kérdezhess az előadóktól!

File: src/components/tiles/mobil-app-tile.tsx
Comment (Tschonti) [inline src/components/tiles/mobil-app-tile.tsx]: sorry, that's my fault, but there is a typo here: it should be 'követhesd', not 'köveshesd'

**Review by Tschonti — CHANGES_REQUESTED** [review]
image
There's no gap between the images on small screens, and the scaling is still a bit weird. I think the appStore should be below the google on small screens, that should solve these problems

File: src/components/tiles/mobil-app-tile.tsx
Comment (FearsomeRover) [reply src/components/tiles/mobil-app-tile.tsx]: Yea, that was not the intended look small screens.

**Review by Tschonti — DISMISSED** [review]
Perfect, but let's not merge it until the app is released!

**Comment by Tschonti** [comment]
öööö az utolsó commit biztos ide akart jönni?

**Comment by Tschonti** [comment]
ooopsiee, shouldn't have done that xd

I'll fix it once it's ready to merge

**Review by Tschonti — APPROVED** [review]

**Pull request merged**
